docs/src/pages/landing-page/QuasarComponents.vue

```diff
@@ -34,24 +42,53 @@
 import { defineComponent, ref, computed } from 'vue'
 import { components } from 'src/assets/landing-page/image-links.js'
 
-const FILTER_CHIPS = [ 'Buttons', 'Inputs', 'Loading', 'Media', 'Navigation', 'Panels & Popups', 'Scroll', 'Tables', 'Others' ]
+const OTHER_CATEGORY = 'other'
+
+const FILTER_CHIPS = [
+  { label: 'Buttons', value: 'button' },
+  { label: 'Inputs', value: 'input' },
+  { label: 'Loading', value: 'loading' },
+  { label: 'Media', value: 'media' },
+  { label: 'Navigation', value: 'navigation' },
+  { label: 'Panels & Popups', value: 'panelAndPopup' },
+  { label: 'Scroll', value: 'scroll' },
+  { label: 'Tables', value: 'table' },
+  { label: 'Others', value: OTHER_CATEGORY }
+]
 
 const componentNameToKebabCase = (componentName) => componentName.replaceAll(' ', '-').toLowerCase()
 
 export default defineComponent({
   name: 'Components',
   setup () {
     const search = ref('')
+    const filterCategory = ref('')
+
+    function filterByCategory (categories, filterCategory) {
+      // If a category was not explicitly set, then we consider it to be of category $OTHER_CATEGORY
+      return typeof categories === 'undefined' ? filterCategory === OTHER_CATEGORY : categories.includes(filterCategory)
+    }
 
-    computed(search, () => {
-      // TODO: implement component filtering
+    const filteredComponents = computed(() => {
+      const searchValue = search.value.toLowerCase()
+      return components.filter(({ name, description, categories }) =>
+        (name.toLowerCase().includes(searchValue) || description.toLowerCase().includes(searchValue)) &&
+        (filterCategory.value === '' || filterByCategory(categories, filterCategory.value))
+      )
     })
 
+    function setFilterCategory (filterChipValue) {
+      // if the filter category is the same as the one we are trying to set, then we reset the filter
+      filterCategory.value = filterCategory.value === filterChipValue ? '' : filterChipValue
+    }
+
     return {
       search,
       filterChips: FILTER_CHIPS,
-      components,
-      componentNameToKebabCase
+      filterCategory,
+      components: filteredComponents,
```

Comment:
Is this renaming intended? I think it's better to just use `filteredComponents` name